pkg/controller/dashboards/netstats.json

```diff
@@ -0,0 +1,1495 @@
+{
+    "__inputs":[
```

Comment:
Is there any way to put a summary/description at the top of these dashboards?

Reply:
Id like to inform the user that OVS/OVN drops arent included here somehow and this is just linux networking stack drops. I dont know how to accomplish this.

Reply:
I don't think we can have a dashboard description text (I can ask the monitoring team) - but how about adding this precision in the drop panels, something like "at interface":
- Current Drops at Interface
- Packets Received Dropped at Interface, by Node
etc.
 ?